### ISSUE TITLE
[Feature request] Allow showing repo history even if clicking button in a file

### ISSUE THREAD

**Issue by sly1061101** [DonJayamanne/gitHistoryVSCode#589]
When clicking ![image](https://user-images.githubusercontent.com/18494936/95381194-6b731280-08ad-11eb-8d98-5ec1b6a1ed9f.png) in a file, it used to show repo history instead of file history. Now it's change in the latest version but I still wanted the old behavior. Could we provide a config option to decide whether to show repo history or file history?

**Comment by zqofnewbie** [comment]
i have the same problem,too.

**Comment by ole1986** [comment]
I hope I can achieve this with the next update. Sorry for that

**Comment by ole1986** [comment]
A new setting will be available with version 0.6.12: 
```
"gitHistory.editorTitleButtonOpenRepo": {
    "type": "boolean",
    "default": false,
    "scope": "window",
    "description": "Prefer to open repository (instead of file) when pressing the editor title button"
}
```

**Comment by void-mAlex** [comment]
this was so confusing until I stumbled upon the changelog saying this was fixed
thank you for introducing the option, but I have to say I really appreciated the previous default
